doc/source/contribution_and_api/unit_testing.rst

```diff
@@ -0,0 +1,42 @@
+.. _ref_unit_testing_contributing:
+
+Unit Testing
+===============
+
+Unit testing is highly important. It enables to check that the
+changes on the code are efficient and don't have an impact on
+other parts of the code. 
+
+Unit tests are in the ``tests`` directory inside this repository.
+
+Test Validation GitHub workflow
+-------------------------------
+
+`Codecov <https://github.com/codecov>`_ generates a coverage report of
+the committed code. It details how merging the Pull Request will impact the
+coverage. It is one of the checks that must be approved to
+merge the code.
+
+Creating a unit test 
+--------------------
+
+In the pymapdl repository, the tests are done with the **pytest package**. 
```

Comment:
```suggestion
In the pymapdl repository, the tests ran using the **pytest package**. 
```
Ad a link to the package too.

Reply:
I would also add some comments on how the unit tests should be. This link is great for this:

https://stackoverflow.com/a/61868/6650211

If you use it, please refer it.

Reply:
Also I would add this info about the arguments of ``assert``. For example this:
```py
assert x == "goodbye", "x should be 'hello'"
```

https://www.w3schools.com/python/ref_keyword_assert.asp

Again, if you use any link, please refer to it. Maybe you can find a better link than the above one too.